src/pages/patientView/genomicOverview/GenomicOverview.spec.tsx

```diff
@@ -1,21 +1,182 @@
-import * as GenomicOverview from './GenomicOverview';
+import GenomicOverview from './GenomicOverview';
 import React from 'react';
 import { assert } from 'chai';
-import { shallow, mount } from 'enzyme';
+import {shallow, mount, ShallowWrapper, ReactWrapper} from 'enzyme';
 import sinon from 'sinon';
+import SampleManager from "../sampleManager";
+import {ThumbnailExpandVAFPlot} from "../vafPlot/ThumbnailExpandVAFPlot";
+import {Mutation} from "../../../shared/api/generated/CBioPortalAPI";
+import {MutationFrequenciesBySample} from "../vafPlot/VAFPlot";
 
 describe('GenomicOverview', () => {
+    let GO:ShallowWrapper<any,any>;
```

Comment:
Can we rename this to `genomicOverview` or something similar? It looks like a constant to me when it is all uppercase :)